### PR TITLE
Add a new spinner design behind a labs flag

### PR DESCRIPTION
**Pull request by anoadragon453**
Video of the new design in action: https://matrix.amorgan.xyz/_matrix/media/r0/download/amorgan.xyz/71953f90530066a42768acd2280bff92851a40d4

Placed behind the labs flag `feature_new_spinner`.

### CODE REVIEW

File: src/components/views/elements/Spinner.js [hunk 1]
```diff
@@ -16,19 +16,39 @@ limitations under the License.
...
-            <div className="mx_Spinner">
-                <img src={require("../../../../res/img/spinner.gif")} width={w} height={h} className={imgClass} />
-            </div>
-        );
-    },
-});
+    return (
+        <div className={divClass}>
+            { message && <React.Fragment><div className="mx_Spinner_Msg">{ message}</div>&nbsp;</React.Fragment> }
+                <img
```
Comment (t3chguy) [inline src/components/views/elements/Spinner.js:37]: over-indented

**Comment by anoadragon453** [comment]
> Looks sane, but doesn't it need a filter invert somewhere for dark mode?

Check the video. I switched the spinner's colour to green which works on light and dark backgrounds :slightly_smiling_face:

**Comment by t3chguy** [comment]
Oh dear, completely missed there was a video :D

**Comment by t3chguy** [comment]
Test fail due to outdated base branch

**Pull request merged**